File: Backend/QuizMaster1/Models/Payment.cs
using System;
using System.Collections.Generic;

namespace QuizMaster1.Models
{
    public partial class Payment
    {
        public int Pid { get; set; }
        public string? Mode { get; set; }
        public int? Amount { get; set; }
        public DateOnly? Date { get; set; }
        public DateOnly? Expirydate { get; set; }
        public int? Sid { get; set; }

        public virtual Student? SidNavigation { get; set; }
    }
}

File: Backend/QuizMaster/quizmaster/QuizMasterSpringBoot/src/main/java/com/quizmaster/entities/Payment.java
package com.quizmaster.entities;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor

@Table(name = "payments")
public class Payment {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int pid;
	@Column
	String mode;
	@Column
	Date date;
	@Column
	Date expirydate;
	
	@OneToOne
	@JoinColumn(name = "sid")
	Student sid;
	

}
